src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -652,6 +781,9 @@ export class HaAutomationEditor extends PreventUnsavedMixin(
   private _valueChanged(ev: CustomEvent<{ value: AutomationConfig }>) {
     ev.stopPropagation();
 
+    // reset the pasted config as soon as the user starts editing
+    this._resetPastedConfig();
```

Comment:
We should only do this when there is a `this._previousConfig` set right? Otherwise we would always dismiss every toast on an edit.